src/wp-includes/class-walker-nav-menu.php

```diff
@@ -73,9 +73,27 @@ public function start_lvl( &$output, $depth = 0, $args = null ) {
 		 * @param int      $depth   Depth of menu item. Used for padding.
 		 */
 		$class_names = implode( ' ', apply_filters( 'nav_menu_submenu_css_class', $classes, $args, $depth ) );
-		$class_names = $class_names ? ' class="' . esc_attr( $class_names ) . '"' : '';
 
-		$output .= "{$n}{$indent}<ul$class_names>{$n}";
+		$atts 		   = array();
+		$atts['class'] = ! empty( $class_names ) ? $class_names : '';
+
+		/**
+		 * Filters the HTML attributes applied to a menu item's ul element.
+		 *
+		 * @since 6.1.2
```

Comment:
```suggestion
		 * @since 6.2.0
```
The earliest that this new filter could be merged to Core is WordPress 6.2.0.

🔢 Applies elsewhere in the PR.